tests/token_bridge_test.cairo

```diff
@@ -339,3 +338,116 @@ fn disable_withdrwal_not_owner() {
     );
 }
 
+#[test]
+fn unblock_token_ok() {
+    let mut mock = mock_state_testing();
+    let usdc_address = deploy_erc20("usdc", "usdc");
+
+    // Setting the token active
+    let old_settings = mock.token_settings.read(usdc_address);
+    mock
+        .token_settings
+        .write(usdc_address, TokenSettings { token_status: TokenStatus::Blocked, ..old_settings });
```

Comment:
so we can directly write to storage in mocks?

Reply:
Oh yes, found a way to do this. This import `use starknet_bridge::bridge::token_bridge::TokenBridge::__member_module_token_settings::InternalContractMemberStateTrait;` allows to do it